packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -201,13 +214,21 @@ export class SOQLEditorInstance {
       const message = `No default org found. Set a default org to use SOQL Builder. Run "SFDX: Create a Default Scratch Org" or "SFDX: Authorize an Org" to set one.`;
```

Comment:
Same, move this message to `src/messages/i18n.ts`

Reply:
I've got this one covered here: https://github.com/forcedotcom/salesforcedx-vscode/pull/2810